integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala

```diff
@@ -860,17 +852,19 @@ case class CarbonDropTableCommand(ifExistsSet: Boolean,
         .dropTable(storePath, identifier)(sparkSession)
       LOGGER.audit(s"Deleted table [$tableName] under database [$dbName]")
     } finally {
-      if (carbonLock != null && isLocked) {
-        if (carbonLock.unlock()) {
+      LOGGER.audit(s"Dropping table $dbName.$tableName failed as the Table is locked")
```

Comment:
This can be possible in positive case as well, both cases needs to be separately logged.